css/index.css

```diff
@@ -1,49 +1,130 @@
-body{
-    background-color:#f2f2f2;
-    font-family: Arial, Helvetica, sans-serif;
-    
+*{
+    box-sizing: border-box;
+    margin: 0px;
+    padding: 0px;
+}
+body {
+    background-color:#f2f2f2ea;
+    font-family: 'Roboto Slab', serif;
+    font-size: 1rem;
+    display: flex;
+    flex-direction: column;
+   
 } 
+/*all text goes to left a little bit*/
 body section p{
 margin-left: 2%;
 }
+/* header and navigation bar*/
 nav{
     background-color: #e1e16b;
+    color: #333333;    
+    display: block;
+    width: 100%;
+    justify-content: left;
+    position: fixed;
```

Comment:
Fix: On devices that are less than 685px, your "About" section is hidden behind the nav bar. I'd recommend adding some top margin or top padding to the About section to mitigate this obstacle.

For example, the About section is readable on a 685px window:
![](https://github.com/ofeterovskaya/fenrir-intro/assets/22607722/ccc6611f-f9a8-4384-a39a-29ad486c3c87)

But becomes cut-off/hidden by the nav when it's a 400px window:
![](https://github.com/ofeterovskaya/fenrir-intro/assets/22607722/c2860bed-468a-4563-8a14-318bcb74928c)